services/workflows-service/src/auth/session-serializer.ts

```diff
@@ -18,14 +19,16 @@ export class SessionSerializer extends PassportSerializer {
   }
 
   async deserializeUser(user: User, done: (err: unknown, user: User | null) => void) {
-    const userResult = await this.userService.getById(user.id);
+    try {
```

Comment:
Otherwise crashes after cleaning the database. `getById` throws an error when a user is not found.